packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoInput/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import PropType from 'prop-types';
+import { CarouselInput, CarouselSlide, CarouselActions } from '@strapi/design-system/CarouselInput';
+import { IconButton } from '@strapi/design-system/IconButton';
+import { Box } from '@strapi/design-system/Box';
+import Plus from '@strapi/icons/Plus';
+
+const LogoInput = ({ customLogo, defaultLogo }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <CarouselInput
+      label={formatMessage({
+        id: 'Settings.application.customization.carousel.title',
+        defaultMessage: 'Logo',
+      })}
+      selectedSlide={0}
+      hint="Change the admin panel logo (Max dimension: 750*750, Max file size: TBC)"
+      previousLabel="Previous slide"
+      nextLabel="Next slide"
+      onNext={() => {}}
+      onPrevious={() => {}}
+      secondaryLabel={customLogo?.name || 'logo.png'}
```

Comment:
Why `logo.png` 🤔 ? (probably a temporary label 😅 )

Reply:
actually it is the label chosen for default Logo 😅 here a screenshot from the mockup and validated with @maevalienard 
<img width="401" alt="image" src="https://user-images.githubusercontent.com/71838159/161023820-01e191aa-73ce-429c-8307-04356bd62f4e.png">

Reply:
ok for me